net/ddns-scripts/files/usr/lib/ddns/update_route53_v1.sh

```diff
@@ -12,8 +12,10 @@ ENDPOINT="route53.amazonaws.com"
 RECORD_TTL=300
 RECORD_NAME="${lookup_host}."
 [ ${use_ipv6} -eq 0 ] && RECORD_TYPE="A"
+[ ${use_ipv6} -eq 0 ] && RECORD_VALUE="${__IP}"
 [ ${use_ipv6} -eq 1 ] && RECORD_TYPE="AAAA"
-RECORD_VALUE="${LOCAL_IP}"
+[ ${use_ipv6} -eq 1 ] && RECORD_VALUE="${__IPV6}"
```

Comment:
This results in an empty `Values` field in the XML. The `__IP` variable has both v4 & v6 (depending on context).

Reply:
Thanks, fixed this one as well!